tools,meta,doc: update text to reflect TSC charter changes (#47126)
nodejs/node: GOVERNANCE.md

## Technical Steering Committee

A subset of the collaborators forms the Technical Steering Committee (TSC).

> The TSC voting members have final authority over this project, including:

Reviewer: ```suggestion
The TSC has final authority over this project, including:
```

Author: 👍

`tsc-agenda` label. That will add it to the TSC meeting agenda.

Before each TSC meeting, the meeting chair will share the agenda with members of

> the TSC. TSC voting members can also add items to the agenda at the beginning of
> each meeting. The meeting chair and the TSC cannot veto or remove items.

Reviewer: There's no reason non-voting members could not do that, right?
```suggestion
the TSC. TSC members can also add items to the agenda at the beginning of each
meeting. The meeting chair and the TSC cannot veto or remove items.
```
